source/player_guide/missions_info/missions/kill_the_traitor.rst

```diff
@@ -9,12 +9,7 @@ Category
 
 .. topic:: Description
 
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
-
-   🚨 PLACEHOLDER 🚨
-
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
-
+Note: You can find Intel on the Traitor’s family from dead enemy soldiers. This can be used by Petros to blackmail the Traitor into staying quiet. 
```

Comment:
To show notes, you can use:
```rst
.. note::
   You can find Intel on the Traitor’s family from dead enemy soldiers. This can be used by Petros to blackmail the Traitor into staying quiet.
```


to indicate a note, this can also be done inside a topic, but everything has to be indented once inside a topic.